cpp/include/cudf_test/column_wrapper.hpp

```diff
@@ -1499,11 +1499,8 @@ class lists_column_wrapper : public detail::column_wrapper {
 
     // concatenate them together, skipping children that are null.
     std::vector<column_view> children;
-    thrust::copy_if(std::cbegin(cols),
-                    std::cend(cols),
-                    valids,  // stencil
-                    std::back_inserter(children),
-                    thrust::identity<bool>{});
+    thrust::copy_if(
+      std::cbegin(cols), std::cend(cols), valids, std::back_inserter(children), thrust::identity{});
```

Comment:
And for this one does it come from the type of `valids` somehow?

Reply:
Correct. The compiler knows the type of `valids`, and for the stencil overload it knows it will be apply the unary predicate `thrust::identity` to that sequence so it is able to deduce.

Reply:
Neat.